src/ApiPort/ApiPort/ApiPort.props

```diff
@@ -3,20 +3,29 @@
     <OutputType>Exe</OutputType>
     <AssemblyName>ApiPort</AssemblyName>
     <RootNamespace>ApiPort</RootNamespace>
-    <!--
-      netcoreapp target must be first to avoid a ResXFileCodeGenerator issue
-      (tracked at https://github.com/dotnet/project-system/issues/1519)
-    -->
-    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
-    <RuntimeIdentifiers>win7-x64;win7-x86</RuntimeIdentifiers>
-    <DefineConstants>$(DefineConstants);FEATURE_SYSTEM_PROXY;FEATURE_NETWORK_CREDENTIAL</DefineConstants>
   </PropertyGroup>
+  <Choose>
+    <!-- build as global tool if GlobalTool is specified-->
+    <When Condition="'$(GlobalTool)' == 'true'">
+      <PropertyGroup>
+      <PackAsTool>true</PackAsTool>
+      <ToolCommandName>ApiPort</ToolCommandName>
+     <!-- <PackageOutputPath>./nupkg</PackageOutputPath> -->
+        </PropertyGroup>
+    </When>
+    <!-- build as standardalone .NET Core app by default-->
+    <Otherwise>
+      <PropertyGroup>
+        <RuntimeIdentifiers>win7-x64;win7-x86</RuntimeIdentifiers>
+        <!--TODO: These constants was defined when building as 461 app, looks out of date. Do we still need to define these constants?
```

Comment:
These aren't needed now AFAIK - it's auto added I believe

Reply:
Looks like FEATURE_SYSTEM_PROXY and FEATURE_NETWORK_CREDENTIAL was defined when targeting 4.6.1 was because  WebRequest.GetSystemWebProxy() and NetworkCredential.SecurePassword were not available on .NET Standard 1.3. They are supported on .NET standard 2.0+. Since this PR removed .NET standard 1.3 support,  looks like it is okay to remove both defineconstants.

Reply:
@twsouthwick Actually. I said too fast, SystemWebProxy.GetProxy throws platformnotsupported exception. This means that the code under ifdef FEATURE_SYSTEM_PROXY still only work on desktop. With this PR, we actually removing desktop support. What would be the customer visible missing functionality on .NET Core then?

Reply:
@twsouthwick added net461 targetframework back to the console tools when building self-contained because FEATURE_SYSTEM_PROXY still only works on desktop, we don't want to regress existing functionality